api-tests/src/clients/cri-stub-client.ts

```diff
@@ -5,6 +5,25 @@ import {
 } from "../types/cri-stub.js";
 import config from "../config/config.js";
 
+const CREDENTIAL_ISSUER_SUBDOMAINS: Record<string, string> = {
+  ticf: "ticf",
+  kbv: "experian-kbv-cri",
+  dcmaw: "dcmaw-cri",
+  address: "address-cri",
+  fraud: "fraud-cri",
+  ukPassport: "passport-cri",
+  drivingLicence: "driving-license-cri",
+  claimedIdentity: "claimed-identity-cri",
+  f2f: "f2f-cri",
+  nino: "nino-cri",
+  dwpKbv: "dwp-kbv-cri",
+  hmrcKbv: "hmrc-kbv-cri",
+  bav: "bav-cri",
+};
```

Comment:
`generateVc` was failing because the cri id didn't match the subdomain of the CRI e.g.
```
| criId                   | scenario           |
| drivingLicence  | some-scenario |
```
so I added some mapping here. Not sure if this is the best approach though as it means it would need to be kept up to date

Reply:
I guess the alternative is just to use the 'mapped' version in the table, but it's a bit weird that it doesn't match our usual `criId` format. This is probably easier to use :)

(cc @Wynndow - unless you had a different idea for how this would work?)

Reply:
Eww. No, sorry, I think this is probably as reasonable as it gets. We're all used to using the criId's we have, but the subdomains don't align. The other option is to fix that...